recipes/sdl_ttf/3.x/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=2.4"
+
+
+class SdlttfConan(ConanFile):
+    name = "sdl_ttf"
+    description = "A TrueType font library for SDL"
+    license = "Zlib"
+    topics = ("sdl3", "sdl3_ttf", "sdl", "ttf", "font")
+    homepage = "https://github.com/libsdl-org/SDL_ttf"
+    url = "https://github.com/conan-io/conan-center-index"
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_harfbuzz": [True, False],
+        "with_plutosvg": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_harfbuzz": True,
+        "with_plutosvg": True,
+    }
+    implements = ["auto_shared_fpic"]
+    languages = "C"
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("freetype/2.13.2")
+        self.requires("sdl/[>=3.2.6 <4]", transitive_headers=True, transitive_libs=True)
+        if self.options.with_harfbuzz:
+            self.requires("harfbuzz/[>=8.3.0]")
+        if self.options.with_plutosvg:
+            self.requires("plutosvg/0.0.7")
```

Comment:
Perhaps we should include CMake as a `tool_require`?

```suggestion
            self.requires("plutosvg/0.0.7")

        def build_requirements(self):
            self.tool_requires("cmake/[>=3.16]")

```